crates/fuzzing/src/generators.rs

```diff
@@ -62,20 +266,19 @@ pub struct WasmtimeConfig {
     memory_config: MemoryConfig,
     force_jump_veneers: bool,
     memfd: bool,
+    /// Configuration for the instance allocation strategy to use.
+    pub strategy: InstanceAllocationStrategy,
 }
 
 #[derive(Arbitrary, Clone, Debug, Eq, Hash, PartialEq)]
 enum MemoryConfig {
     /// Configuration for linear memories which correspond to normal
     /// configuration settings in `wasmtime` itself. This will tweak various
     /// parameters about static/dynamic memories.
-    ///
-    /// Note that we use 32-bit values here to avoid blowing the 64-bit address
-    /// space by requesting ungodly-large sizes/guards.
     Normal {
-        static_memory_maximum_size: Option<u32>,
-        static_memory_guard_size: Option<u32>,
-        dynamic_memory_guard_size: Option<u32>,
+        static_memory_maximum_size: Option<u64>,
+        static_memory_guard_size: Option<u64>,
+        dynamic_memory_guard_size: Option<u64>,
```

Comment:
I'm a little worried about this change in that above there's a `.min(0x1000..)` to I think make the cap still the same but randomly choosing a 64-bit number seems like it'll almost always use the 0x1000.. limit instead of something smaller? Overall I'm wondering if this 64-bit space heavily weights the fuzzing to always choose the same result and whether that has an affect.